src/scripts/constants.ts

```diff
@@ -255,14 +264,18 @@ export module Constants {
 		export var cachedNotebooks = "notebooks";
 		export var currentSelectedSection = "curSection";
 		export var displayLanguageOverride = "displayLocaleOverride";
+		export var doNotPromptRatings = "doNotPromptRatings";
 		export var flightingInfo = "flightingInfo";
+		export var lastBadRatingDate = "lastBadRatingDate";
+		export var lastBadRatingVersion = "lastBadRatingVersion";
 		export var lastClippedDate = "lastClippedDate";
 		export var lastSeenVersion = "lastSeenVersion";
 		export var lastInvokedDate = "lastInvokedDate";
 		export var lastSeenTooltipTimeBase = "lastSeenTooltipTime";
 		export var lastClippedTooltipTimeBase = "lastClippedTooltipTime";
 		export var locale = "locale";
 		export var locStrings = "locStrings";
+		export var numClipSuccess = "numClipSuccess";
```

Comment:
nit and also tedious :stuck_out_tongue:, I feel like `numSuccessfulClips` is a more descriptive name.